azext_iot/dps/providers/aad_oauth.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrest.authentication import Authentication
+
+
+class IoTDPSOAuth(Authentication):
```

Comment:
We don't need another auth module as there are no additional customizations here -- we can probably use the same aad_oauth module across services IoT Hub/DPS/DigitalTwins/Central putting it in a generic location. There is a oneoff auth.py in azext_iot/common used by Central. That particular one could be handled later.